src/code/02_declarations.js

```diff
@@ -80,6 +80,9 @@
                     case '0.89.0':
                         message += 'Check out the new <a href="https://discord.gg/cTH4yaw">Discord channel</a>!<br/><br/>Kissanimelist (v0.89.0)</br>- Add Search to miniMAL</br>- Add MyAnimeList Bookmarks to miniMAL</br>- MyAnimeList Tags don\'t need to be activated anymore</br>- Mal2Crunchyroll links now hides remaining seasons</br>';
                         break;
+                    case '0.90.0':
+                        message += 'Kissanimelist (v0.90.0):<br/>    - Added a shortcut for MiniMAL ( CTRL + M )<br/>    - Added MiniMAL position and dimension settings<br/>    - Added an option for displaying \'Episode Hoverinfo\'<br/>    - Added miniMAL to MyAnimeList<br/>    - Changed the \'Add to Mal\'-message, to a non-blocking message<br/>    - Fix database structure<br/><br/>New on KissAnimelists <a href="https://discord.gg/cTH4yaw">Discord</a>:<br/>    - Feed showing newly added episodes for each of the supported streaming sites.';
```

Comment:
1. You should probably be consistent in how you refer to the script.  
You mention it as both `Kissanimelist` and `KissAnimelist`. It would be better if it was written in the same way everywhere. In my opinion, it should be written as either `kissanimelist` or `KissAnimeList`.  

2. It would probably be better to add the word `changelog` to the first line: `KissAnimeList changelog (v.0.90.0)`  

3. The line `- Fix database structure` sounds like something you are planning to do, it should probably be `- Fixed the database structure` or `- Optimized the database structure` instead.